docs/notebooks/Common_Gotchas_in_JAX.ipynb

```diff
@@ -357,6 +357,45 @@
     "jax_array[1, :] = 1.0"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "8f520bec",
+   "metadata": {},
+   "source": [
+    "And if we try to do `__iadd__`-style in-place updating, we get __different behavior than NumPy__!  (☉_☉)  (☉_☉)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20fbed45",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "jax_array = jnp.array([10, 20])\n",
```

Comment:
It looks like this causes a failure, because a cell further down expects `jax_array` to be two-dimensional.

Maybe re-define `jax_array` again there?

Reply:
Thanks for the catch!